zircon.core/common/src/main/kotlin/org/hexworks/zircon/api/extensions/NumberInputExtensions.kt

```diff
@@ -0,0 +1,14 @@
+package org.hexworks.zircon.api.extensions
+
+import org.hexworks.cobalt.databinding.api.event.ChangeEvent
+import org.hexworks.cobalt.events.api.Subscription
+import org.hexworks.zircon.api.behavior.ChangeListener
+import org.hexworks.zircon.api.component.NumberInput
+
+inline fun NumberInput.onValueChanged(crossinline fn: (ChangeEvent<Int>) -> Unit): Subscription {
```

Comment:
We're not going to need these functions in the future as they have included `Function0`, `Function1`, `FunctionN` in the Kotlin stdlib, but don't worry about this now, I'm working on the refactor on another branch.